src/styles/components/_app_catalog.sass

```diff
@@ -75,43 +75,49 @@
 
 .app-catalog--repos
   display: flex
-  flex-flow: row wrap
+  flex-flow: column
   margin-top: 10px
 
+.app-catalog--description
+  padding: 10px 20px
+  padding-left: 0px
+
+  h3
+    margin-bottom: 0px
+    margin-top: 10px
+
 .app-catalog--repo
-  flex: 0 0 260px
-  margin-right: 25px
-  margin-bottom: 30px
+  flex: 0 0
+  margin-bottom: 15px
   border-radius: 5px
   background-color: #2f556a
+  flex-flow: row
+  display: flex
 
   p
     margin-bottom: 0px
-    padding: 10px 20px
     border-bottom-left-radius: 5px
     border-bottom-right-radius: 5px
+    font-size: 14px
 
   &:hover
     text-decoration: none
     box-shadow: 0px 0px 8px lighten(#1e4156, 22%);
```

Comment:
```suggestion
    box-shadow: 0px 0px 8px lighten(#1e4156, 22%)
```

Unrelated fix

Reply:
Please double-check the white space here. Not sure whether github got this right.